src/ui/server/helpers/summary-lists/policy-and-export/multiple-contract-policy-fields/index.ts

```diff
@@ -0,0 +1,52 @@
+import { FIELDS } from '../../../../content-strings/fields/insurance';
+import { FIELD_IDS } from '../../../../constants';
+import fieldGroupItem from '../../generate-field-group-item';
+import formatCurrency from '../../../format-currency';
+import mapMonthString from '../../../data-content-mappings/map-month-string';
+import { ApplicationPolicyAndExport, SummaryListItemData } from '../../../../../types';
+
+const {
+  INSURANCE: {
+    POLICY_AND_EXPORTS: {
+      CONTRACT_POLICY: {
+        MULTIPLE: { TOTAL_MONTHS_OF_COVER, TOTAL_SALES_TO_BUYER, MAXIMUM_BUYER_WILL_OWE },
+      },
+    },
+  },
+} = FIELD_IDS;
```

Comment:
`FIELD_IDS.INSURANCE.POLICY_AND_EXPORTS`? more readable

Reply:
Linting would complain about that 😄 I've updated it to be more readable - want to use this in other places as well.

Instead of importing root `FIELD_IDS`, just import the ones needed for the file

```js
import FIELD_IDS from '../../../../constants/field-ids/insurance/policy-and-exports';
```